test/cpp/qps/client_async.cc

```diff
@@ -130,39 +130,26 @@ class ClientRpcContextUnaryImpl : public ClientRpcContext {
       response_reader_;
 };
 
-class AsyncUnaryClient GRPC_FINAL : public Client {
+class AsyncClient : public Client {
  public:
-  explicit AsyncUnaryClient(const ClientConfig& config) : Client(config) {
+  explicit AsyncClient(const ClientConfig& config,
+                       void (*setup_ctx)(CompletionQueue*, TestService::Stub*,
```

Comment:
Yes. std::function seems to be allowed in all tests and in the main C++ codebase, so there's no reason not to use it. Will fix.